examples/chainermn/cifar/train_cifar.py

```diff
@@ -0,0 +1,138 @@
+import argparse
+
+import chainermn
+import chainer
+import chainer.links as L
+from chainer import training
+from chainer.training import extensions
+from chainer.training import triggers
+
+from chainer.datasets import get_cifar10
+from chainer.datasets import get_cifar100
+
+import models.VGG
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Chainer CIFAR example:')
+    parser.add_argument('--dataset', '-d', default='cifar10',
+                        help='The dataset to use: cifar10 or cifar100')
+    parser.add_argument('--batchsize', '-b', type=int, default=64,
```

Comment:
Please update the docstring of `--batchsize` to something like "Number of images per GPU in a mini-batch", or "number of images per micro-batch".

Reply:
update "Number of images per GPU in a mini-batch"

Reply:
Update the Number of images per GPU in a mini-batch.